subtest/docs/documentation.md

```diff
@@ -2,111 +2,85 @@
+- [Prerequisites](#prerequisites)
+- [Quick start](#quick-start)
+  * [My first scenario](#my-first-scenario)
+  * [Select a pre-implemented dataset](#select-a-pre-implemented-dataset)
+  * [Set some ML parameters](#set-some-ml-parameters)
+  * [Run it](#run-it)
+  * [Results](#results)
+  * [Contributivity measurement methods](#contributivity-measurement-methods)
+- [Scenario parameters](#scenario-parameters)
+  * [Choice of dataset](#choice-of-dataset)
+  * [Definition of collaborative scenarios](#definition-of-collaborative-scenarios)
+  * [Configuration of the collaborative and distributed learning](#configuration-of-the-collaborative-and-distributed-learning)
+  * [Configuration of contributivity measurement methods to be tested](#configuration-of-contributivity-measurement-methods-to-be-tested)
+  * [Miscellaneous](#miscellaneous)
+- [Dataset generation](#dataset-generation)
+  * [Dataset](#dataset)
+  * [Model generator](#model-generator)
+  * [Preprocessing](#preprocessing)
+  * [Train/validation/test splits](#train-validation-test-splits)
+- [Contacts, contributions, collaborations](#contacts--contributions--collaborations)
+
+## Prerequisites
 
 At root folder:
 ```bash
 pip install -r requirements.txt
-pip install -i https://test.pypi.org/simple/ subtest==0.0.0.6
+pip install -i https://test.pypi.org/simple/ subtest==0.0.0.18
 ```
+The requirements can be found in the subtest folder. The installation will work with the dev-requirements as well, but you will install extra dependencies, as pytest and flake8, which are optional.  
 Note: This is the temporary package for our library.
 
-## Check out our tutorials!
-
-For a better start and a quick understanding of how our library work, we recommend to take a look at our Notebook based tutorials.
-
 ## Quick start
 
-### Important classes:
-
-#### Dataset
-
-This is where you define your dataset and relatives objects such as preprocessing functions and model generator.
-
-##### Dataset
-
-This is the structure of the dataset generator:
+### My first scenario
+To launch a collaborative round, you need to generate scenario and run it, though a Scenario object.
 
+There are 2 mandatory parameters for a collaborative run: `partners_count` and `amounts_per_partner`.
+For instance, you could want to see what is happening with 3 partners, the first with 20% of the total dataset, the second 50% and the third 30% (for a total of 100%).
+Here is an example of how you should do it:
 ```python
-dataset = dataset.Dataset(
-    "name",
-    X_train,
-    X_test,
-    y_train,
-    y_test,
-    input_shape,
-    num_classes,
-    preprocess_dataset_labels,      # See below
-    generate_new_model_for_dataset  # See below
-)
+from subtest.scenario import Scenario
+my_scenario = Scenario(partners_count=3,
+                       amounts_per_partner=[0.2, 0.3, 0.5])
 ```
+Note that you can use more advanced sample split options in order to fine tune the data distribution between partners. See the doc
```

Comment:
It could be nice to point towards a specific section on "set the documentation".